src/IIIF/PresentationAPI/Properties/Thumbnail.php

```diff
@@ -22,12 +22,89 @@
 */
 namespace IIIF\PresentationAPI\Properties;
 
+use IIIF\PresentationAPI\Parameters\Identifier;
 use IIIF\PresentationAPI\Properties\MimeAbstract;
+use IIIF\Utils\ArrayCreator;
 
 /**
  * Implementation of thumbnail descriptive property
  * http://iiif.io/api/presentation/2.1/#thumbnail
  *
  */
 class Thumbnail extends MimeAbstract {
+
+    private $type;
+    private $width;
+    private $height;
+
+  /**
+   * Set the type.
+   *
+   * @param string $type
+   */
+  public function setType($type)
+  {
+    $this->type = $type;
```

Comment:
I am not sure if this can be anything other than ```dcTypes:image```. Would it be worth hardcoding it using the CONST from https://github.com/yale-web-technologies/IIIF-Manifest-Generator/blob/8878521fc7cc32f45bfb39f3bc0a07dc0fc73e5c/src/IIIF/PresentationAPI/Parameters/DCType.php#L31?